tsconfig.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
```

Comment:
We will ideally remove this once we switch `src/index.js` to `src/index.ts` so we can get strict types across the whole package